02 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/00.json

```diff
@@ -2,6 +2,6 @@
   "type" : "landing",
   "heading" : "Market Hours",
   "subHeading" : "",
-  "content" : "<p>Select one of the following Futures markets to see its operating hours:</p>",
```

Comment:
I think we should keep this new heading

Reply:
np. Should be inherited from an old version only